### PR TITLE
Create a new persona: Historian

### PR DESCRIPTION
**Pull request by marcodel13**
### What this PR does
This PR creates a new persona, Travel Historian, which uses the Command model.

### How it was tested
Unit test.
Test on Streamlit. 

### PR checklist

- [x] No API keys or other secrets committed to source?
- [x] New functionality is added alongside appropriate tests?
- [x] All source files have the following header?

```
# Copyright (c) {YEAR} Cohere Inc. and its affiliates.
#
# Licensed under the MIT License (the "License");
# you may not use this file except in compliance with the License.
#
# You may obtain a copy of the License in the LICENSE file at the top
# level of this repository.
```

### CODE REVIEW

**Review by marinamachado — COMMENTED** [review]
I just wrote a small question, otherwise looks good to me! Thanks for this and the experiments that you did:)

File: conversant/personas/historian/config.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "chatbot_config": {
+        "max_context_examples": 10,
+        "avatar": ":amphora:"
+    },
+    "client_config": {
+        "model": "command-52b-v5-dec22-eeayj60s",
+        "max_tokens": 100,
+        "temperature": 0.75,
+        "stop_sequences": ["\nUser"]
```
[the remark this answers is not in the document]
Comment (yichern) [reply conversant/personas/historian/config.json:10]: Yup! `\nUser` is the stop sequence for command models, more discussion [here](https://cohereai.slack.com/archives/C03SVF95DQ9/p1671644551378419?thread_ts=1671644514.709789&cid=C03SVF95DQ9)

**Comment by yichern** [comment]
Thanks for the comprehensive experiments documented on the Notion doc! I had hunches of 
- the importance of role consistency / alignment
- that we are able to prompt command models to have domain grounding (a la character ai)

and I'm glad now there are experiments to back these up! Great work on the historian persona, I think we can transition every other persona to the command model too!

**Review by yichern — DISMISSED** [review]
Thanks for the comprehensive and documented experiments on the Notion doc! I had hunches about
- the importance of a solution enforcing role consistency / alignment
- that we can prompt our command model to have any form of domain grounding easily

and it's good to see these backed up by the experiments!

I think we can move on to transitioning every other persona to a command model too.

File: conversant/personas/historian/config.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+    "chatbot_config": {
+        "max_context_examples": 10,
+        "avatar": ":amphora:"
+    },
+    "client_config": {
+        "model": "command-52b-v5-dec22-eeayj60s",
```
Comment (yichern) [inline conversant/personas/historian/config.json:7]: Can we change this model to `command-xlarge-20221108`, the latest command model that is also in our playground?

**Review by yichern — CHANGES_REQUESTED** [review]
Added a request to change model name!

File: conversant/personas/historian/config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcodel13) [reply conversant/personas/historian/config.json:10]: Yes, the idea was to have this Historian as the first persona using Command. I now plan to go ahead and modify all the others, so to transition from xlarge to command for all the personas: do you guys think it is a good idea?

File: conversant/personas/historian/config.json (hunk 2 above)
Comment (marcodel13) [reply conversant/personas/historian/config.json]: Done!

**Pull request merged**